indico/modules/events/abstracts/templates/abstract/judge.html

```diff
@@ -19,13 +19,13 @@
                             'negative': _("No reviewer proposed to accept this abstract"),
                             'mixed': _("Not all reviewers proposed to accept this abstract")} %}
     {% set state = abstract.reviewing_state %}
-    <span class="i-tag {{ mapping_class[state.name] }}" title="{{ mapping_title[state.name] }}">
+    <span class="i-tag {{ mapping_class[state.name] }} reviewing-state"
```

Comment:
I'd keep the dynamic class at the end and not in the middle of two static classes